docs/src/cli/transfer-tokens.md

```diff
@@ -31,7 +31,15 @@ solana airdrop 1 <RECIPIENT_ACCOUNT_ADDRESS> --url https://api.devnet.solana.com
 ```
 
 where you replace the text `<RECIPIENT_ACCOUNT_ADDRESS>` with your base58-encoded
-public key/wallet address.
+public key/wallet address. 
```

Comment:
ci doesn't like the trailing whitespace added here

Reply:
Fixed! Thanks for your help